tap-gui/plugins/runtime-resource-visibility.md

```diff
@@ -112,33 +98,29 @@ specified in `metadata.ownerReferences`. You can use this section to navigate be
 For more information about owners and dependents, see the
 [Kubernetes documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/owners-dependents/).
 
-![Screenshot of metadata owner references](images/runtime-resources-ownership.png)
+![Screenshot of metadata owner references](images/runtime-resources-ownership-new.png)
```

Comment:
**QUESTION**

Why are all of these named as `*-new` rather than just replacing the original image?